pkg/codegen/testing/test/testdata/simple-enum-schema/schema.json

```diff
@@ -244,11 +244,13 @@
       "namespaces": {
         "plant": "Plant",
         "tree/v1": "Tree.V1"
-      }
+      },
+      "respectSchemaVersion": true
     },
     "go": {
       "importBasePath": "simple-enum-schema/plant",
-      "generateExtraInputTypes": true
+      "generateExtraInputTypes": true,
+      "respectSchemaVersion": true
```

Comment:
So I assume "respectSchemaVersion" is borked on Python and Go because I've turned this on but there's no diff....